lib/aws-record/record/item_operations.rb

```diff
@@ -161,6 +188,15 @@ def update(new_params, opts = {})
       # @option opts [Boolean] :force if true, will save as a put operation and
       #  overwrite any existing item on the remote end. Otherwise, and by
       #  default, will either perform a conditional put or an update call.
+      # @option opts [Hash] :put_item_options Options to pass to the
+      # {http://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/DynamoDB/Client.html#put_item-instance_method
+      # Aws::DynamoDB::Client#put_item} call. This option is only used when
```

Comment:
This is fairly confusing and pushes the responsibility of knowing if its a new item or an existing item to the user.... I know why we're doing it since the supported options between those calls differ.  But, it seems difficult to use (unless you are setting `force`).   Could we combine them and support only the intersection of options from put and update?

Reply:
Yeah.. that's very true. I think we could take one set of opts and just try them? Param validator would catch cases where something in update would not exist in put or vice versa. That could be slightly annoying in some rare cases.. otherwise validation might be weird? We could potentially get members from each of those operation inputs and validate that way.

Reply:
Changed to flat options and it's used in both cases.